### ISSUE TITLE
Material Articles list pagination not keyboard navigable

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4549]
Found while testing for icon a11y.  The pagination is not keyboard focusable at the bottom of the /articles/ page listing all articles.

The links to pages show up as links in voiceover rotor, but they are skipped when tabbing, and I've tried tabbing through the whole page without managing to land on them.  The content is read out with Voiceover too, so this seems to be keyboard navigation issue only.

template is `elements/pagination.html`.

Expected behaviour:
I should be able to tab into page links after tabbing out of the final keyboard focusable item within an article card. 

<img width="699" alt="image" src="https://github.com/user-attachments/assets/702d21ec-f4d2-47f2-bd93-c9551e7baa5f" />

**Comment by mauromsl** [comment]
Leaving it until the end of the sprint in case there is time. But priority is Clean theme
